certbot_route53/authenticator.py

```diff
@@ -1,5 +1,6 @@
 """Route53 Let's Encrypt authenticator plugin."""
 import logging
+from six.moves import xrange
```

Comment:
Rather than depending on six, since we can materialize the range in memory for py2, I'd rather we just use `range`.